### ISSUE TITLE
Error Parsing Schema

### ISSUE THREAD

**Issue by johnthepink** [prisma/upgrade#54]
Running `npx prisma-upgrade` with the following schema errors out:

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource postgresql {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model Test {
  Interchange Interchange[] @relation("Interchanges", references: [id])
}
```

This throws the error:

```
SyntaxError: Expected ".", [ \t\n\r], or [_A-Za-z0-9] but "[" found.
  at peg$buildStructuredError (/Users/john/code/steve/node_modules/prismafile/dist/print.js:664:12)
  at Object.peg$parse [as parse] (/Users/john/code/steve/node_modules/prismafile/dist/print.js:2992:11)
  at Object.parse (/Users/john/code/steve/node_modules/prismafile/dist/print.js:3008:17)
  at new Schema (/Users/john/code/steve/node_modules/prisma-upgrade/dist/index2.js:17:25)
  at main (/Users/john/code/steve/node_modules/prisma-upgrade/dist/cli/index.js:197:15)
```

Based on trial and error, it seems to be ending up in code that thinks `Interchange[]` is an `Int`. Similarly this schema throws the same error thinking it is a `String`:

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource postgresql {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model Test {
  Interchange StringCheese[] @relation("Interchanges", references: [id])
}
```

We have types that use the word `Interchange`, so this is currently blocking us. Any thoughts on how to get around it? Thanks!

**Comment by matthewmueller** [comment]
Thanks so much for raising this issue. It should be fixed in 0.0.31. It's a good case study in why you should keep semantics separate from syntax.

**Comment by johnthepink** [comment]
@matthewmueller thanks! that did the trick.
